ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+cluster_profile=/var/run/secrets/ci.openshift.io/cluster-profile
+secret_dir=/tmp/secret
+
+export SSH_PRIV_KEY_PATH=${cluster_profile}/ssh-privatekey
+export PULL_SECRET_PATH=${cluster_profile}/pull-secret
+export SSHOPTS="-o ConnectTimeout=5 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ServerAliveInterval=90 -i ${SSH_PRIV_KEY_PATH}"
+
+echo "************ baremetalds devscripts setup command ************"
+env | sort
+
+# Initial check
+if [ "${CLUSTER_TYPE}" != "packet" ] ; then
+    echo >&2 "Unsupported cluster type '${CLUSTER_TYPE}'"
+    exit 1
+fi
+
+echo "-------[ $SHARED_DIR ]"
+ls -ll ${SHARED_DIR}
+
+# Fetch packet server IP
+IP=$(cat ${SHARED_DIR}/server-ip)
+export IP
+echo "Packet server IP is ${IP}"
+
+# Applying NSS fix for SSH connection and share artifacts
```

Comment:
This shouldn't be necessary in the libvirt container. It already contains (*wink*) the necessary files.